interfaces/builtin/network_setup_control.go

```diff
@@ -45,6 +45,18 @@ const networkSetupControlConnectedPlugAppArmor = `
 
 /run/udev/rules.d/ rw,                 # needed for cloud-init
 /run/udev/rules.d/[0-9]*-netplan-* rw,
+
+capability net_admin,
+
+#include <abstractions/dbus-session-strict>
+
+# Allow use of NetPlan Apply API, used to apply network configuration
+dbus (send)
+    bus=system
+    interface=io.netplan.Netplan
+	path=/io/netplan/Netplan
```

Comment:
The spacing here is a bit odd, tabs-vs-spaces?

Reply:
yes I will fix this up